LLama/Experimental/Core/LLamaCpp/LLamaGreedySamplingMethod.cs

```diff
@@ -0,0 +1,41 @@
+﻿using LLama.Experimental.Abstractions;
+using LLama.Experimental.Common;
+using LLama.Native;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LLama.Experimental.Core.LLamaCpp
+{
+    // TODO: This is only the most simple implementation to run the example. It should be replaced in the future.
+    public class LLamaGreedySamplingMethod: ISamplingMethod
```

Comment:
(I think I may have mentioned this already in another reply, but I can't find it now).

This looks like the same as the [`GreedySamplingPipeline`](https://github.com/SciSharp/LLamaSharp/blob/master/LLama/Sampling/GreedySamplingPipeline.cs). And more generally I guess `ISamplingMethod` is basically the same as `ISamplingPipeline`?

Reply:
Yes, they are very similar. I renamed it only because I don't want to change the current implementations because of some slight difference.